### ISSUE TITLE
How can I concatenate 2d and 3d tensors?

### ISSUE THREAD

**Issue by Eddie-Hwang** [Mathux/ACTOR#9]
[xseq = torch.cat((self.muQuery[y][None], self.sigmaQuery[y][None], x), axis=0)](https://github.com/Mathux/ACTOR/blob/c7237e82e333bf2c57f7d8e12f27d0831233befc/src/models/architectures/transformer.py#L115)

This code tries to concatenate 2d and 3d tensors.
Can you explain how it works?

Thanks in advance.

**Comment by Qwinpin** [comment]
How did you solve that?
